tests/integration_tests/performance/configs/test_snap_restore_performance_config_4.14.json

```diff
@@ -2182,350 +2182,350 @@
+                                                    "target": 13.748,
+                                                    "delta_percentage": 19
                                                 }
                                             }
                                         },
                                         "3vcpu_128mb": {
                                             "P50": {
                                                 "restore": {
-                                                    "target": 7.385,
-                                                    "delta_percentage": 15
+                                                    "target": 13.533,
+                                                    "delta_percentage": 14
                                                 }
                                             },
                                             "P90": {
                                                 "restore": {
-                                                    "target": 7.881,
-                                                    "delta_percentage": 29
+                                                    "target": 13.952,
+                                                    "delta_percentage": 15
                                                 }
                                             }
                                         },
                                         "4vcpu_128mb": {
                                             "P50": {
                                                 "restore": {
-                                                    "target": 7.493,
-                                                    "delta_percentage": 14
+                                                    "target": 13.879,
+                                                    "delta_percentage": 16
                                                 }
                                             },
                                             "P90": {
                                                 "restore": {
-                                                    "target": 8.006,
-                                                    "delta_percentage": 22
+                                                    "target": 14.281,
+                                                    "delta_percentage": 16
                                                 }
                                             }
                                         },
                                         "5vcpu_128mb": {
                                             "P50": {
                                                 "restore": {
-                                                    "target": 7.605,
-                                                    "delta_percentage": 15
+                                                    "target": 14.283,
+                                                    "delta_percentage": 17
```

Comment:
This is quite a big increase. Let's decouple adding missing baselines from the updates and investigate this latency bump.

Reply:
That's right. As this PR aims to add missing performance baselines for AMD (m6a.metal), let's decouple! Thanks!